customer-churn/requirements.txt

```diff
@@ -0,0 +1,9 @@
+feature_engine==1.2.0
+imbalanced_learn==0.9.0
+imblearn==0.0
+lightgbm==3.3.2
+statsmodels
+xgboost
+zenml==0.7.3
+facets-overview
```

Comment:
Why is facets a dependency?

Reply:
I think I tried to use facets explicitly in the past so for that reason while generating requirements.txt it added facets as well, but Thanks for pointing out, I will remove it

Reply:
I did this in another PR :)